node/plugin/cmd/main.go

```diff
@@ -117,6 +124,21 @@ func pluginOps(ctx *cli.Context) {
 		}
 	}
 
+	if config.Operation == plugin.OperationListQuorums {
+		operatorAddress, err := tx.OperatorIDToAddress(context.Background(), operatorID)
+		if err != nil {
+			log.Printf("Error: failed to get operator address for operatorID: %x, error: %v", operatorID, err)
+			return
+		}
+		quorumIds, err := tx.GetRegisteredQuorumIdsForOperator(context.Background(), operatorID)
+		if err != nil {
+			log.Printf("Error: failed to get quorum(s) for operatorID: %x, operator address: %x, error: %v", operatorID, sk.Address, err)
```

Comment:
isn't `sk` nil here?

Reply:
ah yes. I fixed the other logs but missed this one

Reply:
cca159bf8fd31d7356f648cc6908bcdae59a2ca7